RadixWallet/Features/AssetsFeature/Components/HideAsset/HideAsset.swift

```diff
@@ -0,0 +1,120 @@
+// MARK: - HideAsset
+@Reducer
```

Comment:
If the macro is causing issues when combined with FeatureReducer, feel free to skip the latter, and implement the needed actions etc manually.